app/js/systemoverview.js

```diff
@@ -0,0 +1,128 @@
+    var sessionId = BfsSystemOverview.getSessionId();
+    BfsSystemOverview.getOrgLimits(sessionId);
+}
+
+BfsSystemOverview.getSessionId = function () {
+    return document.cookie.match("sid=([^;]*)")[1];
+}
+
+BfsSystemOverview.getOrgLimits = function (sessionId) {
+    $.ajax({
+        url: '/services/data/v40.0/limits/',
+        headers: { 'Authorization': 'Bearer ' + sessionId }
+    })
+    .always(BfsSystemOverview.processOrgLimitsResponse);
+}
+
+BfsSystemOverview.processOrgLimitsResponse = function (data, textStatus, jqXHR) {
+    if (textStatus == "success") {
+        BfsSystemOverview.showOrgLimitsUI(data);
+    } else {
+        console.log(jqXHR)
+    }
+}
+
+BfsSystemOverview.showOrgLimitsUI = function (orgLimits) {
+    var orgLimitsUI = '<div class="panel-container"><span><div class="panel">';
+    orgLimitsUI += '<div class="top-line"><h2>Other Org Limits </h2><img src="https://github.com/mattsimonis/boostr/blob/master/app/icon.png?raw=true" style="height:32px; width:32px; padding-left: 20px;" title="Provided by Boostr for Salesforce" /></div><div class="content"><div class="panelContent">';
+
+    var alreadyShownLimits = ['DailyApiRequests', 'DataStorageMB', 'FileStorageMB'];
+    var numberFormatter = new Intl.NumberFormat();
+
+    Object.keys(orgLimits).forEach(function(limitName, index) {
+        if (alreadyShownLimits.indexOf(limitName) != -1) {
+            return;
+        }
+
+        var limitInfo = BfsSystemOverview.LimitInfoMap[limitName];
+        var limitRemaining = orgLimits[limitName].Remaining;
+        var limitMax = orgLimits[limitName].Max;
+        var limitUsed = limitMax - limitRemaining;
+        var percentageUsed = Math.round((limitUsed / limitMax) * 100);
+        var percentageAvailable = 100 - percentageUsed;
+
+
+        var borderTopClass = 'border-top';
+        var panelWarningClass = '';
+        var barPositiveWarnClass = '';
+
+        if (index == 0) {
+            borderTopClass = '';
+        }
+
+        if (percentageUsed >= 80) {
+            panelWarningClass = 'usage-warn';
+            barPositiveWarnClass = 'bar-positive-warn';
+        }
+
+        orgLimitsUI += '<div class="panel-content-item ' + borderTopClass + ' ' + panelWarningClass + '">';
+
+        orgLimitsUI += '<div class="panelLeft">';
+
+        orgLimitsUI += '<div class="type"><span class="title">' + limitInfo.Title +
+                       '</span>';
+
+        if (limitInfo.Description) {
+            orgLimitsUI += '<div class="mouseOverInfoOuter" onfocus="addMouseOver(this)" onmouseover="addMouseOver(this)" tabindex="0">'
+                              + '<img src="/img/s.gif" alt class="infoIcon" title>'
+                              + '<div class="mouseOverInfo" style="display:none; opacity: -0.2; left: 21px;">'
+                                  + '<div class="body">' + limitInfo.Description + '</div>'
+                              + '</div>'
+                        + '</div>';
```

Comment:
No particular reason. Just thought it would be more "intention revealing" this way to see the markup created. Is there a compelling reason to do it the other way?